layouts/projects/summary.html

```diff
@@ -0,0 +1,48 @@
+<div class="project-summary clearfix">
+	<div about="{{ .Permalink | relURL }}">
+		<div class="col-xs-6 col-md-4">
+			<a href="{{ .Permalink | relURL }}" >
+				<img class="img-responsive center-block" src="{{ .Params.logo }}" alt="{{ .Title }} logo" />
+			</a>
+		</div>
+
+		<div class="col-xs-18 col-md-14 project-teaser">
+			<h2 class="name">{{ .Title }}</h2>
+			{{ if isset .Params.technology_types 0 }}
+				<ul class="labels list-inline">
+					{{range .Params.technology_types }}
+			            <li><a class="label label-warning" href="/{{ "technology_types" | urlize }}/{{ . | urlize}}/">{{ . }}</a></li>
+					{{ end }}
+				</ul>
+			{{ end }}
+			<div class="project-teaser-description">
+				{{ .Content }} 
+				<p><a href="{{ .Permalink | relURL }}">Read more...</a></p>
+			</div>
+		</div>
+
+		<div class="col-md-6 details hidden-xs hidden-sm">
+			<p class="latest-release">
+				Latest release: <a href="/projects/modeling.mmt.qvt-oml/releases/2019-06-3.9.3"><span class="badge version">0.0</span></a>
+			</p>
+
+			<div class="btn-group">
+				<a href="{{ .Permalink | relURL }}#downloads" class="btn btn-sm btn-warning details-link">Download</a>
+				<button type="button" class="btn btn-sm btn-warning dropdown-toggle details-link" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+					<span class="caret"></span> <span class="sr-only">Toggle Dropdown</span>
+				</button>
+				<ul class="dropdown-menu">
```

Comment:
I think we can remove the dropdown for now unless you want to create a page parameter for each link. I think for now we can live with only the download link.